lib/src/main/java/com/cloudbees/groovy/cps/CpsTransformer.java

```diff
@@ -763,6 +763,29 @@ public void run() {
         BINARY_OP_TO_BUILDER_METHOD.put(KEYWORD_IN, "isCase");
     }
 
+    private void multipleAssignment(final Expression parentExpression,
+                                    TupleExpression tuple,
+                                    ListExpression list) {
+        List<Expression> listExpressions = list.getExpressions();
+        List<Expression> tupleExpressions = tuple.getExpressions();
+        if (listExpressions.size() < tupleExpressions.size()) {
```

Comment:
And if it is greater, then what? Is that legal?

Reply:
I'm aping code in Groovy itself - need to think about it more, though, 'cos I may want to error on it.